### DISCUSSION TITLE
Any way to get all selected rows out of component?

### DISCUSSION THREAD

**Discussion by lozynskiadam** [Power-Components/livewire-powergrid#1270] [Q&A] [answered]
Is there any possibility to retrieve selected rows on all powergrid pages using separate livewire component or javascript?

**Comment by abhaypithadiya** [comment, marked as answer]
Hey!

You can use ```window.pgBulkActions.get(your_table_name)``` to retrieve selected rows in Javascript.

If you have named your table you can change the table name or it will be 'default'. 

Hope this helps!

**Reply by lozynskiadam** [reply]
it worked, thanks!
